stl/inc/xthreads.h

```diff
@@ -33,19 +33,23 @@ struct _Stl_critical_section {
 };
 
 struct _Mtx_internal_imp_t {
-#ifdef _CRT_WINDOWS
+#if defined(_CRT_WINDOWS) || defined(UNDOCKED_WINDOWS_UCRT)
+#pragma detect_mismatch("_CRT_WINDOWS_mutex_size", "1")

```

Comment:
`_CRT_WINDOWS_mutex_size` seems a bit hefty. How about `windows_mutex_abi`, or just `windows_mutex` given that all detect_mismatch uses are fundamentally about ABI?